src/features/identityCheck/pages/errors/EduConnectErrors.tsx

```diff
@@ -5,9 +5,18 @@ import { EduConnectError } from 'features/identityCheck/errors/eduConnect/types'
 import { EduConnectErrorMessageEnum } from 'features/identityCheck/errors/hooks/useNotEligibleEduConnectErrorData'
 import { UseRouteType } from 'features/navigation/RootNavigator'
 
+const allowedDomains = [
+  'https://educonnect.education.gouv.fr',
```

Comment:
Ne faut-il pas échapper les domaines, comme tu avais fait ici : https://github.com/pass-culture/pass-culture-main/commit/6807b7ca8f5226530fc20e1e837468a931d8dd36 ?

```suggestion
  'https://educonnect\.education\.gouv\.fr',
```

Reply:
J'ai vérifié et non, non seulement le linting l'empêche mais ceci est fait au moment ou on fait `new RegEx`, exemple via ta console JS, tu peux coller ça  :

```js
var url = 'https://educonnect.education.gouv.fr'
var re = new RegExp(`^${url}`)
console.log(re)
console.log(re.test('https://educonnect.education.gouv.fr'))
console.log(re.test('https://educonnecteducationgouv.fr'))
```

Sinon si tu veux tester la regex : https://regex101.com/r/oGQ7qv/1

Les rajouter n'est pas gênant, exemple:


```js
var url = 'https:\/\/educonnect\.education\.gouv\.fr'
var re = new RegExp(`^${url}`)
console.log(re)
console.log(re.test('https://educonnect.education.gouv.fr'))
console.log(re.test('https://educonnecteducationgouv.fr'))
```